index.js

```diff
@@ -217,10 +218,12 @@ module.exports = function makeFetch (opts = {}) {
           return new FakeResponse(statusCode, 'ok', responseHeaders, stream, url)
         }
       } else {
+        responseHeaders.set('Allow', 'GET, HEAD, PUT, DELETE')
         return new FakeResponse(405, 'Method Not Allowed', responseHeaders, intoStream('Method Not Allowed'), url)
       }
     } catch (e) {
-      return new FakeResponse(500, 'server error', responseHeaders, intoStream(e.stack), url)
+      const status = (e.message === NOT_WRITABLE_ERROR) ? 403 : 500
```

Comment:
If it's a 403 then it should return some text like "Hyperdrive not writeable" instead of the stacktrace